src/VisualStudio/Core/Def/Implementation/Preview/FileChange.cs

```diff
@@ -79,13 +80,18 @@ private ChangeList ComputeChildren(TextDocument left, TextDocument right, Cancel
 
             diffService = diffService ?? diffSelector.DefaultTextDifferencingService;
 
-            var diff = ComputeDiffSpans(oldText, newText, diffService, cancellationToken);
-            if (!diff.Differences.Any())
+            var diff = ComputeDiffSpans(diffService, left, right, cancellationToken);
+            if (diff.Differences.Count == 0)
             {
-                // There are no changes. Just show the whole document.
-                return GetEntireDocumentAsSpanChange(left);
+                // There are no changes.
+                return ChangeList.Empty;
```

Comment:
I think we've gone back and forth on this now--there was a time when no differences would show you the black box and no checkboxes. Oh well.